libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardLayout/utils/sizing.ts

```diff
@@ -131,12 +134,42 @@ function implicitLayoutItemSizeFromXlSize(xlSize: IDashboardLayoutSize): IDashbo
  */
 function dashboardLayoutItemSizeForAllScreens(
     heightAsRatio: number,
+    gridHeight: number,
     xl: number,
     lg: number,
     md: number,
     sm: number,
     xs: number,
 ): IDashboardLayoutSizeByScreenSize {
+    if (gridHeight) {
+        return {
+            xl: {
+                gridWidth: xl,
+                heightAsRatio,
```

Comment:
Should we propagate `heightAsRatio` if there is `gridHeight`?

Reply:
Dont think so, `gridHeight` has higher priority. Will remove `heightAsRatio` from here.